submissions/yaripey/html-css-popup/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
+  <title>Popup task</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <a href="#" class="nav-item">Mail</a>
```

Comment:
would be better to add the markup `ul > li` for main menu

Reply:
Should it be `header > ul > li`?